src/KeyVault/KeyVault/Models/PSKeyVaultCertificatePolicy.cs

```diff
@@ -328,6 +746,21 @@ internal static PSKeyVaultCertificatePolicy FromCertificatePolicy(CertificatePol
             return intValueGetter(lifetimeAction.Trigger);
         }
 
+        private static int? FindIntValueForEmailAction(IList<Track2Certificate.LifetimeAction> lifetimeActions)
+        {
+            var lifetimeAction =
+                lifetimeActions.FirstOrDefault(x => string.IsNullOrEmpty(x.Action.ToString()) && 0 == string.Compare(x.Action.ToString(), Track2Certificate.CertificatePolicyAction.EmailContacts.ToString(), true)
```

Comment:
```suggestion
                lifetimeActions.FirstOrDefault(x => x.Action?.ToString()?.Equals(Track2Certificate.CertificatePolicyAction.EmailContacts.ToString(), StringComparison.OrdinalIgnoreCase);
```
I guess you mean x.Action is not null and empty

Reply:
CS0023	Operator '?' cannot be applied to operand of type 'CertificatePolicyAction'